### ISSUE TITLE
new false positive alert since version '5.2.0'

### ISSUE THREAD

**Issue by MahatmaFatalError** [spotbugs/spotbugs-gradle-plugin#1015]
The following snipped
```java
assertThrows(ConstraintViolationException.class, () -> {
				GenericEventObjectValidator.throwIfInvalid(new Object());
			});
```
from a junit test leads to a `Exception created and dropped rather than thrown ( RV_EXCEPTION_NOT_THROWN )` (IMHO false positive) finding since gradle plugin version id 'com.github.spotbugs' version '5.2.0'.

java 17
gradle-7.6.3

**Comment by hazendaz** [comment]
we released 4.8.1 of spotbugs, we will get gradle plugin out in coming days.

**Comment by KengoTODA** [comment]
The latest release depends on SpotBugs 4.8.2 by default, so closing this issue as resolved.
